Test/Integration/Export/BlueprintExportDataTest.php

```diff
@@ -49,46 +49,12 @@ public function getBlueprintCsvContent(): array
-            8 => 'Name',
-            9 => 'Alias',
-            10 => '"Sku pattern"',
-            11 => '"Title pattern"',
-            12 => '"Test Configurable First (Configurable)"',
-            13 => '"Test Configurable First (Synchronized)"',
-            14 => '"Test Configurable Second (Configurable)"',
-            15 => '"Test Configurable Second (Synchronized)"',
-            16 => '"Test Configurable First"',
-            17 => 'test_configurable_first',
-            18 => '{{sku}}-{{content_vars[\'test_configurable_first\'][\'value\']}}',
-            19 => '{{title}}-{{content_vars[\'test_configurable_first\'][\'value_label\']}}',
-            20 => 'yes',
-            21 => 'no',
-            22 => 'no',
-            23 => 'no',
-            24 => '"Test Configurable Second"',
-            25 => 'test_configurable_second',
-            26 => '{{sku}}-{{content_vars[\'test_configurable_second\'][\'value\']}}',
-            27 => '{{title}}-{{content_vars[\'test_configurable_second\'][\'value_label\']}}',
-            28 => 'no',
-            29 => 'no',
-            30 => 'yes',
-            31 => 'no',
-            32 => '"Test Configurable First Test Configurable Second"',
-            33 => 'test_configurable_first-test_configurable_second',
-            34 => '{{sku}}-{{content_vars[\'test_configurable_first\'][\'value\']}}-{{content_vars[\'test_configurable_second\'][\'value\']}}',
-            35 => '{{title}}-{{content_vars[\'test_configurable_first\'][\'value_label\']}}-{{content_vars[\'test_configurable_second\'][\'value_label\']}}',
-            36 => 'yes',
-            37 => 'no',
-            38 => 'yes',
-            39 => 'no',
+            0 => 'path://name,path://alias,path://sku_pattern,path://title_pattern,path://attribute.encoded__' . $this->base36Coder->encode('sk_color') . '.is_configurable,path://attribute.encoded__' . $this->base36Coder->encode('sk_color') . '.is_synchronized,path://attribute.encoded__' . $this->base36Coder->encode('sk_size') . '.is_configurable,path://attribute.encoded__' . $this->base36Coder->encode('sk_size') . '.is_synchronized,path://attribute.encoded__' . $this->base36Coder->encode('sk_shoe_size') . '.is_configurable,path://attribute.encoded__' . $this->base36Coder->encode('sk_shoe_size') . '.is_synchronized',
+            1 => 'Name,Alias,"Sku pattern","Title pattern","Storekeeper Color (Configurable)","Storekeeper Color (Synchronized)","Storekeeper Size (Configurable)","Storekeeper Size (Synchronized)","Storekeeper Shoe Size (Configurable)","Storekeeper Shoe Size (Synchronized)"',
+            2 => "Storekeeper Color,sk_color,{{sku}}-{{content_vars['sk_color']['value']}},{{title}}-{{content_vars['sk_color']['value_label']}},yes,no,no,no,no,no",
+            3 => "Storekeeper Size,sk_size,{{sku}}-{{content_vars['sk_size']['value']}},{{title}}-{{content_vars['sk_size']['value_label']}},no,no,yes,no,no,no",
```

Comment:
```suggestion
```
Those 2 should not be there (if I read [configurable_products_with_two_attributes.php](https://github.com/storekeeper-company/magento2-plugin/blob/c56ead742f486852a7ee28cdb952824745df1953/Test/Integration/_files/configurable_products_with_two_attributes.php) correct), because there is no configurable product only having single attributes as configurable.